src/jit/gentree.h

```diff
@@ -2538,10 +2570,27 @@ struct GenTreeCall final : public GenTree
     // Returns true if the call has retBuf argument
     bool HasRetBufArg() const { return (gtCallMoreFlags & GTF_CALL_M_RETBUFFARG) != 0; }
 
+    //-----------------------------------------------------------------------------------------
+    // HasMultiRegRetVal: whether the call node returns its value in multiple return registers.
+    //
+    // Arguments:
+    //     None
+
+    // Return Value:
+    //     True if the call is returning a multi-reg return value. False otherwise.
+    //
+    // Note:
+    //     This is implemented only for x64 Unix and yet to be implemented for
+    //      arm64/arm32/x86
+    bool HasMultiRegRetVal() const 
+    { 
 #ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
-    // Returns true if the call is returning a multi-reg struct value
-    bool HasMultiRegRetVal() const { return varTypeIsStruct(gtType) && !HasRetBufArg(); }
+        return varTypeIsStruct(gtType) && !HasRetBufArg(); 
```

Comment:
This is a member method of GenTreeCall and hence no need to assert that the gentree node is GT_CALL